docs/source/support/release-notes.rst

```diff
@@ -9,202 +9,200 @@ This page contains release notes for the SDK.
 HEAD — ongoing
 --------------
 
-- Sandbox: Transactions with a record time that is after the maximum record time (as provided in the original command)
-  are now properly rejected instead of committed to the ledger: `#987 <https://github.com/digital-asset/daml/issues/987>`_
+- **Sandbox**: Transactions with a record time that is after the maximum record time (as provided in the original command) are now properly rejected instead of committed to the ledger.
+
+  See `issue #987 <https://github.com/digital-asset/daml/issues/987>`__ for details.
```

Comment:
I'd just simply put a link to the ticket on the same line without the `for details`. The fact you are linking to issues pretty much says all you need.

Reply:
Yeah - I'd actually like to discourage doing this in general (because it seems to connect up to people not really writing the release note properly), so minimising it makes sense